pkg/exporter/sumologicexporter/translate_metadata.go

```diff
@@ -42,12 +42,16 @@ var metadataTranslations = map[string]string{
 	"k8s.replicaset.name":     "replicaset",
 	"k8s.statefulset.name":    "statefulset",
 	"service.name":            "service",
+	"file.path.resolved":      "_sourceName",
 }
 
 // translateMetadata renames metadata keys according to metadataTranslations.
 func translateMetadata(attributes pdata.AttributeMap) {
 	attributes.Range(func(otKey string, value pdata.AttributeValue) bool {
 		if sumoKey, ok := metadataTranslations[otKey]; ok {
+			if _, ok := attributes.Get(sumoKey); ok {
+				return true
+			}
```

Comment:
Can we add some comment here? It's not obvious to me what this does.

For all attributes, for all all metadata translated attributes check if particular sumo metadata key is in the metric attributes and if it is then skip it (don't remove it?)?

Reply:
I added comment

Reply:
`// do not rename attribute if target name already exists`

Reply:
Sorry but I don't understand: `do not rename attribute if target name already exists` -> this checks if `sumoKey` is in `attributes`. Without this check we simply overwrite the value under this key (and remove the corresponding OT key).

Maybe this should be using a different wording? If it exists do not change the value of already existing Sumo key and do not remove a corresponding OT key ?

Reply:
we can use different wording, I understand it that way.

```
			attributes.Insert(sumoKey, value)
			attributes.Delete(otKey)
```
renames (move) attribute from `otKey` to `sumoKey`

if attribute with name `sumoKey` (target) already exists do not overwrite it